mmdata.c

```diff
@@ -1170,6 +1203,34 @@ nmbrString *nmbrSpace(long n)
   return (sout);
 }
 
+/* Search for the nth occurrence of string2 in string1 */

```

Comment:
Explain parameters (semantics and limitations) in a comment.  For example, what does occ mean? n would match the functions name.